test/e2e/index.test.ts

```diff
@@ -34,7 +34,7 @@ async function attemptGoto(page: playwright.Page, url: string): Promise<boolean>
 }
 
 describe('e2e', () => {
-  const baseUrl = 'http://localhost:5000';
+  const baseUrl = 'http://localhost:5001';
```

Comment:
I something conflicting with the port?

Reply:
Yes, it's in the commit message and you should have had a notification already for https://github.com/mui-org/material-ui/pull/25860#pullrequestreview-640870390